src/views/layouts/_angulardirectives.php

```diff
@@ -126,9 +126,7 @@
                     <?= Admin::t('layout_select_file'); ?>
                 </span>
             </div>
-            <span class="link-selector-reset" ng-click="reset()" ng-show="fileinfo!=null">
-                <i class="material-icons">remove_circle</i>
-            </span>
+            <button type="button" class="btn btn-danger" ng-click="reset()" ng-show="fileinfo!=null" ><?= Admin::t('aws_groupauth_th_remove');?></button>
```

Comment:
As I said in the other PR - Use `btn btn-delete btn-icon` instead of `btn btn-danger`

Reply:
i  will try again as you said，thanks

Reply:
@TheMaaarc Although I also think I changed the look better

Reply:
@baqianxin I like the shadows as well, it looks good. but we should be consistent according to bootstrap 4 - sometimes this is more important

Reply:
@baqianxin For us consistency and the close styles to bootstrap 4 without changing base css is important for us. This way you will be able to apply a bootstrap4 theme to the admin you complet change the look and feel!